.travis.yml

```diff
@@ -0,0 +1,4 @@
+sudo: false
+language: node_js
```

Comment:
@aiwenar It seems that moving the `package.json` into the typeset directory has caused Travis to fail. I can try moving it back to the root and changing paths in `package.json` to point to the `typeset/` directory.